tests/e2e/v3_curl_maxstream_test.go

```diff
@@ -162,22 +150,49 @@ func submitConcurrentWatch(cx ctlCtx, number int, wgDone *sync.WaitGroup, errCh
 	var wgSchedule sync.WaitGroup
 	wgSchedule.Add(number)
 	for i := 0; i < number; i++ {
-		go func(i int) {
+		go func(i int) (err0 error) {
+			defer func() {
+				if err0 != nil {
+					cx.t.Fatalf("testV3CurlMaxStream watch failed: %d, error: %v", i, err0)
+				}
+				wgDone.Done()
+			}()
+
+			cluster := cx.epc
+			member := cluster.Procs[rand.Intn(cluster.Cfg.ClusterSize)]
+			curlReq := e2e.CURLReq{Endpoint: "/v3/watch", Value: string(watchData)}
+
+			args := e2e.CURLPrefixArgs(cluster.Cfg, member, "POST", curlReq)
+			proc, err := e2e.SpawnCmd(args, nil)
+			if err != nil {
+				return fmt.Errorf("failed to spawn: %w", err)
+			}
+			defer proc.Stop()
```

Comment:
It seems not good to copy the detailed implementation from the framework, but we do not have better way for now.

Reply:
The `e2e.SpawnWithExpect` doesn't work well in this case. And the `SpawnWithExpect` is just a helper to write case eaiser. But the `e2e.SpawnCmd` is also exported as helper, right? I was thinking that it is ok to use it 😂